source-google-ads/source_google_ads/custom_query_stream.py

```diff
@@ -17,12 +17,14 @@ def __init__(self, config, **kwargs):
     @property
     def primary_key(self) -> str:
         """
-        The primary_key option is disabled. Config should not provide the primary key.
+        The primary_key option is disabled. Config should not
+        provide the primary key.
         It will be ignored if provided.
-        If you need to enable it, uncomment the next line instead of `return None` and modify your config
+        If you need to enable it, uncomment the next line instead of
+        `return None` and modify your config
         """
-        # return self.config.get("primary_key") or None
-        return None
+        return self.config.get("primary_key") or None
```

Comment:
Can you update the comment above this line, since changing this code has invalidated it?

Reply:
Done!